src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -3,15 +3,121 @@
  */
 import axios from "axios";
 
+const URL = `${window.TL.BASE_URL}analysis/ajax/`;
+
+/*
+ * Get all the data required for the analysis -> details page.
+ * @param {number} submissionId Submission ID
+ * @return {Promise<*>} `data` contains the OK response and the details map;
+ *                      `error` contains error information if an error occurred.
+ */
+export async function getVariablesForDetails(submissionId) {
+  const res = await axios.get(`${URL}details/${submissionId}`);
+  return res.data;
+}
+
+/*
+ * Get analysis input files
+ * @param {number} submissionId Submission ID
+ * @return {Promise<*>} `data` contains the OK response and input files data;
+ *                      `error` contains error information if an error occurred.
+ */
+export async function getAnalysisInputFiles(submissionId) {
+  const res = await axios.get(`${URL}inputs/${submissionId}`);
+  return res.data;
+}
+
+/*
+ * Updates user preference to either receive or not receive an email on
+ * analysis error or completion.
+ * @param {number} submissionId Submission ID
+ * @param {boolean} emailPipelineResult True or False
+ * @return {Promise<*>} `data` contains the OK response; error` contains error information if an error occurred.
+ */
+export async function updateAnalysisEmailPipelineResult(
+  submissionId,
+  emailPipelineResult
+) {
+  const res = await axios.patch(`${URL}update-email-pipeline-result/`, {
+    analysisSubmissionId: submissionId,
+    emailPipelineResult: emailPipelineResult
+  });
+  return res.data;
+}
+
+/*
+ * Updates analysis name and/or analysis priority.
+ * @param {number} submissionId Submission ID
+ * @param {string} analysisName Name of analysis
+ * @param {string} priority [LOW, MEDIUM, HIGH]
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+
```

Comment:
Remove space between JsDoc and method

Reply:
Fixed in f8844707a5f60a885ba87a349edb62b076eaba15